### PR TITLE
[New] Add react-grid-layout@0.16.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #11648 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/STRML/react-grid-layout
 * Official website (optional, not the repository): https://strml.github.io/react-grid-layout/examples/0-showcase.html
 * NPM package url (optional): https://www.npmjs.com/package/react-grid-layout
 * License and its reference: [MIT](https://github.com/STRML/react-grid-layout/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 130
   - Count of stars: 4,524
   - Count of forks: 505
 * NPM download stats (optional):
   - Downloads in the last day: 1,808
   - Downloads in the last week: 11,229
   - Downloads in the last month: 50,497

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
1. Choose the git repo as the source instead of npm because there are more versions on the git repo.
2. Use the rule "**/react-grid-layout*" when the `basePath` is `dist` because we don't need simple-include.html in v0.8.2 ~ v0.9.2.
3. We need style.css as https://github.com/STRML/react-grid-layout#installation said.
4. use `.gitignore` to exclude the unpackaged versions.

**Comment by sufuf3** [comment]
Just a reminder, add [New] in the title is better.

**Comment by extend1994** [comment]
ping @maruilian11 @pvnr0082t

**Comment by extend1994** [comment]
ping @maruilian11 @pvnr0082t

**Comment by sufuf3** [comment]
ping @maruilian11 @pvnr0082t

**Comment by extend1994** [comment]
@maruilian11 
They were minified but in wrong commit. Thanks for review. Now it's updated!
Please help me review this, thank you.

File: .gitignore [hunk 1]
```diff
@@ -217,6 +217,16 @@ node_modules/
...
+/ajax/libs/react-grid-layout/0.0.2
+/ajax/libs/react-grid-layout/0.1.*
+/ajax/libs/react-grid-layout/0.2.*
+/ajax/libs/react-grid-layout/0.3.*
+/ajax/libs/react-grid-layout/0.4.*
+/ajax/libs/react-grid-layout/0.5.*
+/ajax/libs/react-grid-layout/0.6.*
+/ajax/libs/react-grid-layout/0.7.*
+/ajax/libs/react-grid-layout/0.8.0
+/ajax/libs/react-grid-layout/0.11.2
```
Comment (PeterDaveHello) [inline .gitignore]: I think we can combine these patterns?

**Comment by extend1994** [comment]
@cdnjs/intern3 @cdnjs/library-reviewer @cdnjs/active-contributor I update the `.gitignore` according to the request. 
There is a new part: manual adding of react-grid-layout@0.11.2. The details is in the commit msg. Please help me review this PR again, thank you all.

**Comment by sufuf3** [comment]
@extend1994 I know you mention you built v0.11.2 but as this mention https://github.com/cdnjs/cdnjs/pull/11162#issuecomment-299638501 and https://github.com/cdnjs/cdnjs/pull/11162#issuecomment-299665067 I think we need to discuss with the author first to ask the author provide it.
If @PeterDaveHello  says: let's build it, otherwise I think we don't have to do this.
What do you think?

Thank you.

**Comment by extend1994** [comment]
Oh I don't know about that, I can open an issue to ask the author now.

**Comment by extend1994** [comment]
@sufuf3 I directly remove 0.11.2 part. The author said it's broken version in the issue. Please help review it, thanks.

**Comment by extend1994** [comment]
@maruilian11 I added the minifying info in commit msg, please help review it, thanks.

**Comment by extend1994** [comment]
Update with v0.16.1.

**Pull request merged**